src/webui/www/private/css/Layout.css

```diff
@@ -18,231 +18,242 @@ Required by:
 /* Layout
 ---------------------------------------------------------------- */
 
-html, body {
-	background: #fff;
+html,
+body {
+    background: #fff;
 }
 
 body {
-	margin: 0; /* Required */
+    margin: 0;
```

Comment:
Is this line break intended?

Reply:
None of the formatting tools can handle inlined comments, I think I'll revert changes to css files.